subsys/bluetooth/host/conn.c

```diff
@@ -893,9 +893,10 @@ static int start_security(struct bt_conn *conn)
 	{
 		if (!conn->le.keys) {
 			conn->le.keys = bt_keys_find(BT_KEYS_LTK_P256,
-						     &conn->le.dst);
+						     conn->id, &conn->le.dst);
```

Comment:
Could we be consistent and use the same pattern for all calls? (ID as first argument).

Reply:
The pattern I've tried to follow is that the identity precedes remote addresses, so in that sense this is consistent. The general idea is that a remote address no longer uniquely identifies a remote address, rather the tuple (local identity, remote address) must be used. Because of this I'd keep these two parameters together in the parameter lists.

Reply:
Ok.